import-export-cli/integration/testutils/api_testUtils.go

```diff
@@ -759,3 +766,20 @@ func ValidateChangeLifeCycleStatusOfAPIFailure(t *testing.T, args *ApiChangeLife
 	assert.NotContains(t, output, "state changed successfully!", "Error while changing life cycle of API")
 	assert.NotEqual(t, args.Api.LifeCycleStatus, args.ExpectedState, "Life Cycle State changed successfully")
 }
+
+func ValidateGetDevPortalAPIs(t *testing.T, api *apim.API, client *apim.Client, username, password, apiState string) {
+	t.Helper()
+
+	devPortalApisList := getDevPortalAPIs(client, username, password)
+	if devPortalApisList.Count < 1 {
+		base.WaitForIndexing()
+		devPortalApisList = getDevPortalAPIs(client, username, password)
+	}
```

Comment:
```suggestion
	base.WaitForIndexing()
	devPortalApisList = getDevPortalAPIs(client, username, password)
```

Reply:
Shall we directly call base.WaitForIndexing() and get the DevPortal APIs?

Reply:
Also, the base.WaitForIndexing() is already inside the getDevPortalAPIs function as well. Check whether we need to duplicate calling base.WaitForIndexing() in there and here both.

Reply:
Added the changes with https://github.com/wso2/product-apim-tooling/pull/799/commits/5a4905c297e38286b3ad11d381cde936b474eaed

yes, we need this in both places to prevent failing tests intermittently. It wasn't enough the base.WaitForIndexing() inside the getDevPortalAPIs function.